src/ashley/models.py

```diff
@@ -197,3 +212,50 @@ class Meta:
 
 # Default implementation of the AbstractLTIContext model
 LTIContext = model_factory(AbstractLTIContext)
+
+
+def image_directory_path(instance, filename):
+    """File will be uploaded to specific path"""
+    return "{0}_{1}/{2}/{3}".format(
+        instance.forum.id, instance.forum.slug, instance.poster.id, filename
+    )
+
+
+class AbstractUploadImage(DatedModel):
+    """Abstract model for images."""
+
+    file = models.ImageField(
+        upload_to=image_directory_path,
+        max_length=100,
+        blank=True,
+        validators=[
+            validate_upload_image_file_size,
+            validate_upload_image_allowed_type,
+        ],
+        null=True,
+    )
+    forum = models.ForeignKey(
+        "forum.Forum",
```

Comment:
```suggestion
        get_model("forum", "Forum"),
```

With this change (+ the correct import of `get_model`), you can remove your `pylint: disable-all` comment at the beginning of this module.

The error you had (`No module named forum.models`) is related to the forum Foreign key.
The forein key checker of pylint-django doesn't like strings, it seems to prefer class references.